File: day20/homework/classwork1.py
score = int(input("Enter number from 1 to 100: "))

if score in range(90,101):
    print("Your grade is A")
elif score in range(80, 90):
    print("Your grade is B")
elif score in range(70, 80):
    print("Your grade is C")
elif score in range(60, 70):
    print("Your grade is D")
else:
    print("Your grade is F")

File: day71/classwork/classwork1.js
window.onload = function() {
    let num = Math.round(Math.random() * 100)
    let user_num = parseInt(prompt("Enter any number: "));

    function guess_word() {
        if (user_num > num) {
            console.log("Your number is bigger.");
        } else if (user_num < num) {
            console.log("Your number is smaller.");
        } else {
            console.log("You guessed the number!");
        }
    }

    guess_word();
};